src/erldigest.erl

```diff
@@ -2,7 +2,8 @@
 
 -export([calculate_response/5,
          validate_response/5,
-         generate_challenge/2]).
+         generate_challenge/2,
+         get_A1_hash/2]).
```

Comment:
Since this generates the hash and does not just get a pre-calculated value I would change the name to calculate.
Get implies constant time access.

Reply:
Or generate